build/goblin.js

```diff
@@ -3764,13 +3771,13 @@ Goblin.HingeConstraint.prototype.update = (function(){
 
 		// 3,4 are rotational, constraining motion orthogonal to axis
 		world_axis.findOrthogonal( t1, t2 );
-		this.rows[3].jacobian[3] = -t1.x;
-		this.rows[3].jacobian[4] = -t1.y;
-		this.rows[3].jacobian[5] = -t1.z;
+		// this.rows[3].jacobian[3] = -t1.x;
```

Comment:
Can't see these lines in source files. Is it something you didn't want to commit?

Reply:
It's the built file diff, has been changed back and forth (source is at HingeConstraint.js)